.circleci/config.yml

```diff
@@ -679,6 +679,16 @@ jobs:
   # -------------------------
   test_ios_template:
     executor: reactnativeios
+    parameters:
+      flavor:
+        type: string
+        default: "Debug"
+      newarchitecture:
+        type: string
+        default: "OldArch"
+      hermes:
+        type: string
+        default: "Hermes"
```

Comment:
would it be possible to add a flipper or not (since I think flipper in release mode had a problem recently, and release mode issues are typically not discovered until later in the RC cycle after the first few build checks)

And of course my favorite we add a use_frameworks check (with possibly both linkages - so it's actually a tri-state, no frameworks, frameworks+dynamic, frameworks+static)

These are the variations I'm currently aware of

I expect failure in hermes+frameworks+dynamic, and flipper+frameworks+any and newarch+frameworks+any, but on the good side, the other permutations are all working I think

Reply:
As discussed offline (but leaving the reply here for future doc), yes.

I plan to add the Flipper Yes/No, excluding all the `WithFlipper+Release` because flipper is not added by default in Release.

For use framework, yes... it is possible, but requires some more changes also in the podfiles... 

I'll probably add them in a separate PR, though. Just not to add too many things at once.

Reply:
Totally agree with incremental improvement in all things, and specifically here. But as a brainstorm for a future PR: I also just remembered macCatalyst builds. That was an odyssey getting them working again after they broke in 0.63, would be nice to have a canary here that would tell us if they broke again.